app/upgrades.go

```diff
@@ -57,17 +57,16 @@ func (app UmeeApp) RegisterUpgradeHandlers(bool) {
 }
 
 func (app *UmeeApp) registerUpgrade5_1(upgradeInfo upgradetypes.Plan) {
-	planName := "v5.1"
+	planName := "v5.1-beta" // TODO: decide about the final release
```

Comment:
Should we call this "v5.1" regardless (or "v5.1.0" if we need it to be different) and if canon-3 needs two releases, only give the second one a special name?